server/src/main/java/com/cloud/network/NetworkModelImpl.java

```diff
@@ -2673,6 +2673,11 @@ public List<String[]> generateVmData(String userData, String userDataDetails, St
             vmData.add(new String[]{METATDATA_DIR, CLOUD_DOMAIN_ID_FILE, domain.getUuid()});
         }
 
+        String customCloudName = VirtualMachineManager.CloudInitMetadataCloudName.valueIn(datacenterId);
+        if (!StringUtils.isBlank(customCloudName)) {
```

Comment:
```suggestion
        if (StringUtils.isNotBlank(customCloudName)) {
```

Reply:
Sorry @GutoVeronezi , probably better to keep it this way than import an alternate StringUtils class or replace everything with it. I have to rollback this suggestion.

Reply:
Sure, no problem. The refactoring can be done later.